scapy/layers/tls/handshake.py

```diff
@@ -307,7 +311,6 @@ def tls_session_update(self, msg_str):
             for e in self.ext:
                 if isinstance(e, TLS_Ext_SupportedVersion_CH):
                     s.advertised_tls_version = e.versions[0]
-
```

Comment:
Is this change needed?

Reply:
This was just for consistency. It's written elsewhere without the space. Also it's weird